_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
 |  **Performance**         | Not quantified or guaranteed     | Not quantified or guaranteed         | Perf (latency/scale) is quantified, documented, with guarantees against regression 
 |   **Deprecation Policy**        | None     | Weak - 3 months         | Dependable,  Firm. 1 year notice will be provided before changes
 
-## Istio 0.3.0 features status
+## Istio 0.4 features
 
 Below is our list of existing features and their current phases. This information will be updated after every monthly release.
 
-### Config / APIs
-
-
-| Feature           | Phase      
-|-------------------|-------------------
-| [Mesh-wide Settings](https://istio.io/docs/reference/config/service-mesh.html) | Alpha
-| [Basic config validation](https://github.com/istio/istio/issues/1894)    | Alpha
-
-
-### Networking
+### Traffic Management
 
 | Feature           | Phase      
 |-------------------|-------------------
-| [Protocols HTTP 1.1](https://github.com/istio/istio/blob/master/pilot/model/service.go#L104)         | Beta
-| [Distributed Tracing Zipkin / Jaeger](https://istio.io/docs/tasks/telemetry/distributed-tracing.html)        | Beta
-| [Ingress Controller](https://istio.io/docs/tasks/traffic-management/ingress.html)         | Alpha
-| [Basic Egress](https://istio.io/docs/tasks/traffic-management/egress.html)               | Alpha
-| [Protocols HTTP 2.0](https://github.com/istio/istio/blob/master/pilot/model/service.go#L101)        | Alpha
-| [Protocols gRPC](https://github.com/istio/istio/blob/master/pilot/model/service.go#L97)     		   | Alpha
-| [Protocols TCP](https://github.com/istio/istio/blob/master/pilot/model/service.go#L107)         	   | Alpha
+| [Memquota Implementation and Integration](https://istio.io/docs/tasks/telemetry/metrics-logs.html) | Beta
```

Comment:
Can we move this down to the bottom of the list? It's the least interesting thing, yet it's listed as our first feature to the whole product :-)

Reply:
I have listed beta features on top of each table so it's easy to identify them

Reply:
Maybe we don't bother marking this one beta then...it has some issues (e.g. can't run Mixer HA and use it). Would prefer not to highlight.

Reply:
This is not even traffic management.. Typo? Did you mean Metrics, Tracing, Logs, ?

Reply:
Quota / Rate limiting could either go under security or traffic management, we flipped a coin and chose traffic management. I'm fine moving it to security if we think it goes there.

Or we introduce the "Operations" section but we were waiting till we had more than one item to put in there.